packages/jaeger-ui-components/src/ScrollManager.tsx

```diff
@@ -218,7 +218,7 @@ export default class ScrollManager {
    * `setAccessors` is bound in the ctor, so it can be passed as a prop to
    * children components.
    */
-  setAccessors = (accessors: Accessors) => {
+  setAccessors = (accessors: Accessors | TNil) => {
```

Comment:
Do we need the TNil here?

Reply:
We pass `null` to this function in two tests, here's one example: https://github.com/grafana/grafana/blob/82c4c35d179fc47fb2f63fcf9b35ee0c635ed270/packages/jaeger-ui-components/src/ScrollManager.test.ts#L74

So `setAccessors` needs to be able to accept an argument of type `null`. Unless there's a better solution?

Reply:
Well just set something else in the tests. The tests aren't TS so some things work and it was probably easier to do it that way but that does not mean it should be part of the API. Here for example I assume you could create new manager, not set the Accessors and it would test the same thing but with API that probably makes a bit more sense.

Reply:
Got it, not using `setAccessors` is the same thing as not setting accessors so there is no need for this line.
See https://github.com/grafana/grafana/pull/58661/commits/2b4f3b5c27be22523d87e3866d4341057db6d223 and https://github.com/grafana/grafana/pull/58661/commits/aa1cc9bc68e21b27df144adc3e844acca1415ff0.